packages/go_router/lib/src/configuration.dart

```diff
@@ -538,15 +538,35 @@ class RouteConfiguration {
   }
 
   void _debugFullPathsFor(List<RouteBase> routes, String parentFullpath,
-      int depth, StringBuffer sb) {
-    for (final RouteBase route in routes) {
+      String parentDecoration, StringBuffer sb) {
+    for (final (int index, RouteBase route) in routes.indexed) {
+      final String decoration =
+          _getDecoration(parentDecoration, index, routes.length);
+      String path = parentFullpath;
       if (route is GoRoute) {
-        final String fullPath = concatenatePaths(parentFullpath, route.path);
-        sb.writeln('  => ${''.padLeft(depth * 2)}$fullPath');
-        _debugFullPathsFor(route.routes, fullPath, depth + 1, sb);
+        path = concatenatePaths(parentFullpath, route.path);
+        final String screenName =
+            route.builder?.runtimeType.toString().split('=> ').last ?? '';
```

Comment:
Wouldn't this always be `Widget`?

maybe just use `GoRoute` and `GoRoute(redirectOnly)`

Reply:
screenName is the name of the widget. it will not always print `Widget`. Otherwise it was for no use. 
Just like in the `logging_test.dart` test it logs `/ (Text)`

Example from my app:
```
[GoRouter] Full paths for routes:
           ├─/ (SplashScreen)
           ├─/intro (IntroScreen)
           ├─ (ShellRoute)
           │ ├─/main/year ()
           │ │ └─/main/year/month (HomeScreen)
           │ └─/main/profile (ProfileScreen)
           │   ├─/main/profile/analytics (AnalyticsScreen)
           │   └─/main/profile/settings (SettingsScreen)
           │     └─/main/profile/settings/license (LicenseAgreementScreen)
           └─/journal (JournalsListScreen)
             ├─/journal/new (JournalCreateScreen)
             └─/journal/:id (JournalDetailScreen)
               └─/journal/:id/edit (JournalEditScreen)
``` 

There are edge cases: 
- screenName is only get printed, when user uses `GoRouter.builder`
- It does not print for `GoRouter.pageBuilder`
- It prints `(Never)` when `GoRouter.builder` throws `Unimplemented Error`